packages/devtools_shared/lib/src/extensions/extension_model.dart

```diff
@@ -104,14 +108,21 @@ class DevToolsExtensionConfig implements Comparable {
   /// See https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/icons.dart.
   final int materialIconCodePoint;
 
+  /// Whether this extension is distrubuted in a public package on pub.dev.
+  final bool isPublic;
+
   String get displayName => name.toLowerCase();
 
+  /// The extension name that will be used in analytics.
```

Comment:
nit: I would change this method name to make the method comment unneeded. E.g. 
analyticsSafeName